protocol/protobuf/communities.proto

```diff
@@ -50,6 +50,7 @@ message CommunityDescription {
   string outro_message = 12;
   bool encrypted = 13;
   repeated string tags = 14;
+  map<string, CommunityPermission> community_permissions = 15;
```

Comment:
Hmm, We will have now CommunityPermissions and CommunityPermission. Is it possible to merge them some day ? If not maybe we should think about better naming for CommunityPermissions (older one).

Reply:
yes I forgot to ask that question here... 
@jrainville @endulab @cammellos we already have `Permissions` in Community, but it in fact its `Access`. Should I change it here or leave it as it is?